string.md

```diff
@@ -7,19 +7,22 @@ toc: true
 
 The `STRING` [data type](data-types.html) stores a string of characters.
 
-In CockroachDB, the following are aliases for `STRING` and `STRING(n)`: 
+In CockroachDB, the following are aliases for `STRING`: 
 
 - `CHARACTER`
-- `CHARACTER(n)`
 - `CHAR` 
-- `CHAR(n)` 
 - `VARCHAR`
-- `VARCHAR(n)` 
 - `TEXT`
 
+And the following are aliases for `STRING(n)`:
+
+- `CHARACTER(n)`
+- `CHAR(n)` 
```

Comment:
I'd prefer to leave `CHAR` and `VARCHAR` with the same behavior. Let's document it as such, perhaps calling out the lack of padding for `CHAR(n)`.